src/BlueprintServiceProvider.php

```diff
@@ -20,9 +20,17 @@ class BlueprintServiceProvider extends ServiceProvider implements DeferrableProv
     public function boot()
     {
         if (!defined('STUBS_PATH')) {
-            define('STUBS_PATH', dirname(__DIR__) . '/stubs');
+            $stubPath = base_path('stubs/blueprint');
```

Comment:
Is `stubs` the path where Laravel _would_ save to?

Reply:
that's correct.